docs/a2a-firewall.md

```
@@ -0,0 +1,261 @@
+- Special pseudo-header `(request-target)`:
+
+`(request-target): <lower(method)> <path>[?<rawQueryString>]`
+
+### 7.2 Joining
+
+- Join lines with `\n`.
+- No trailing newline.
+
+Example:
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)